### DISCUSSION TITLE
Default built-in postcss plugins dependency error in lastest nextjs 9.3.4

### DISCUSSION THREAD

**Discussion by gtolarc** [vercel/next.js#11580] [Help]
After updating to 9.3.4, the following dependency error occurred. As far as I know, plugins like postcss-flexbugs-fixes are built into nextjs by default, but should I explicitly install them now?

```
> Ready on http://localhost:3000
[ ready ] compiled successfully
[ event ] build page: /
[ wait ]  compiling ...
[ error ] ./assets/global.scss
Error: Loading PostCSS Plugin failed: Cannot find module 'postcss-flexbugs-fixes'
```

**Comment by timneutkens** [comment]
You should definitely install them as they're dependencies of your projects indeed, eg if this is removed from Next.js or replaced it would break your app.

**Reply by gtolarc** [reply]
https://nextjs.org/docs/advanced-features/customizing-postcss-config

The documentation says that the built-in css support includes postcss and the default settings include postcss-flexbugs-fixes. Why did you guys change it to work only if user installed it separately? It seems to be confusing not only for me.

**Reply by timneutkens** [reply]
It only mentions that the default list of plugins is the ones mentioned. Still means you have to install the dependencies when you use them (eg what if Next.js changes dependencies). We can probably add that to the documentation though.

**Reply by gtolarc** [reply]
Especially for beginners, I think the nuances of the document need to be revised. Thanks for the answer.
